libs/shared/package.json

```diff
@@ -39,6 +39,7 @@
   },
   "dependencies": {
     "@clerk/clerk-sdk-node": "^5.0.9",
+    "@nestjs/swagger": "^7.1.8",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** How did this even work with the import missing? I'm guessing some other package pulled it in :/

Reply:
Correct. It's because I moved header utils into shared.